### PR TITLE
V4 updates

### PR DESCRIPTION
**Pull request by donnen**

### CODE REVIEW

File: app/code/community/Apruve/ApruvePayment/Block/Adminhtml/Sales/Order/View.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/Apache-2.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@apruve.com so we can send you a copy immediately.
+ *
+ * @category   Apruve
+ * @package    Apruve_Payment
+ * @copyright  Copyright (coffee) 2014 Apruve, Inc. (http://www.apruve.com).
```
Comment (eiwi1101) [inline app/code/community/Apruve/ApruvePayment/Block/Adminhtml/Sales/Order/View.php]: Mmmm coffeee. Should this not be also 2017?

File: LICENSE [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014, Apruve
+Copyright (c) 2016, Apruve
```
Comment (eiwi1101) [inline LICENSE]: Psssh, this is SO last year.
Also we should have a yearly (c) party!

File: app/code/community/Apruve/ApruvePayment/Model/Api/Abstract.php [hunk 3]
```diff
@@ -66,47 +168,220 @@ protected function _getApiKey()
...
+        }
+    }
+
+    /**
+     * Convert price to needed value
+     * As current version supports only USD, convert price to cents
+     * @param float $price
+     * @return float
+     */
+    protected function convertPrice($price)
```
Comment (eiwi1101) [inline app/code/community/Apruve/ApruvePayment/Model/Api/Abstract.php:233]: Does this convert to the `amount_cents` like value we know and love? If so, shouldn't we do some rounding?

**Review by eiwi1101 — COMMENTED** [review]
Comments about dates, the rest seems... okay? I think? If it works...?

File: app/code/community/Apruve/ApruvePayment/Model/Api/Abstract.php (hunk 3 above)
Comment (donnen) [reply app/code/community/Apruve/ApruvePayment/Model/Api/Abstract.php:233]: Won't a dollar value * 100 always be a round number?

**Comment by eiwi1101** [comment]
Depends on how the dollar value is calculated, I suppose. Since we're
starting with floats and this might potentially process tax, we could get
fractions of cents in the calculation. If we're working with pure dollar
values though and not doing any math beforehand, this should be fine.

**Pull request merged**